DEVELOPMENT.md

```diff
@@ -38,6 +38,6 @@ Documentation files in `docs` are generated by combining:
 Any edits to documentation files should be made in the `templates` directory or the .go files. Then you can use the following steps to generate the files in the docs directory:
 
 1. Download `tfplugindocs` from https://github.com/hashicorp/terraform-plugin-docs/releases and make it available on your `PATH`
-2. Run `$ make docs` or `$ tfplugindocs` from the root of this directory
```

Comment:
Running tfplugindocs directly doesn't work anymore because tfplugindocs builds the provider with the `registry.terraform.io` prefix, but then calls the terraform binary which is now open-tofu that looks for plugins with the prefix of `registry.opentofu.org`.